client/image_load.go

```diff
@@ -12,12 +13,18 @@ import (
 // ImageLoad loads an image in the docker host from the client host.
 // It's up to the caller to close the io.ReadCloser in the
 // ImageLoadResponse returned by this function.
-func (cli *Client) ImageLoad(ctx context.Context, input io.Reader, quiet bool) (types.ImageLoadResponse, error) {
+func (cli *Client) ImageLoad(ctx context.Context, input io.Reader, opts types.ImageLoadOptions) (types.ImageLoadResponse, error) {
 	v := url.Values{}
 	v.Set("quiet", "0")
-	if quiet {
+	if opts.Quiet {
 		v.Set("quiet", "1")
 	}
+	refsJSON, err := json.Marshal(opts.Refs)
+	if err != nil {
+		return types.ImageLoadResponse{}, err
+	}
+	v.Set("refs", string(refsJSON))
+	v.Set("name", opts.Name)
```

Comment:
Looks like this adds `name`, `refs` options to `POST /images/load`; these should be documented in the [swagger.yml](https://github.com/moby/moby/blob/master/api/swagger.yaml), and in the [api version history](https://github.com/moby/moby/blob/master/docs/api/version-history.md)

The `refs` option looks like it needs some documentation (what's the format, and how is it used?); IIUC, it's a mapping from `image:tag`(s) to `name`(s) (OCI)?

Reply:
I'll add doc around that

Reply:
is there a swagger/api version doc for experimental as well? (this is experimental)

Reply:
as discussed on Slack, no real need for swagger API for experimental